tests/bcl-test/BCLTests/tvOS-monotouch_corlib_xunit-test.dll.ignore

```diff
@@ -0,0 +1,9 @@
+# device failures:
+
+# Expected: 0
+# Actual:   -1 
+System.IO.Tests.FileInfo_Exists.TrueForNonRegularFile
+System.IO.Tests.DirectoryInfo_Exists.FalseForNonRegularFile
```

Comment:
there was an issue we found, long time ago, inside tvOS wrt files (symlink?)
might be somewhere in bugzilla...

Reply:
I'll try to find it and add it to github and in the comment